cypress/fixtures/newPetHomePage.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Well done on these fixtures, Ethan! Figuring out the flow of the app, from re-homing the pet and uploading the image, to what the new data set-up would be after posting and creating the fixture for each of those flows, very savvy.